sample-app/tests/browser/testoperator.js

```diff
@@ -0,0 +1,57 @@
+const { BrowserPageWidths } = require("./constants");
+
+/**
+ * Responsible for browser navigation based on given test instructions.
+ */
+class TestOperator {
```

Comment:
The `Operator` suffix makes it sound a bit like this class performs some kind of operations on the tests, rather than just running them. How about `TestRunner`? Does that accurately describe the class's purpose?

Reply:
yea, sounds good